### ISSUE TITLE
Invisible hook when opening chat on older ddnet server

### ISSUE THREAD

**Issue by AssassinTee** [ddnet/ddnet#5893]
When holding the hook on the ceiling, locking it with f1 and then opening chat the hook goes invisible and you get prediction errors (shaky screen).
I tested this on newer DDNet servers as well where everything works properly and this might be an issue with the Unique servers specifically.
I can't pinpoint the versions, since it's not my server and you can't get the server version with /credits.

![screenshot_2022-09-28_14-38-30](https://user-images.githubusercontent.com/821401/192780630-e2d401bf-cc6c-46d9-88b4-5e7f0d8ac4f2.png)

I am hooking the top in this image

**Comment by VoxelDoesCode** [comment]
So I did some tests, and I gathered some info. I think in Unique servers when you type, the hook retracts, but since that's not part of the physics now you're still in place. Note that when you leave chat, you "hook" where you look at, but instantly the hook teleports back to where it was needed. 

This only happens to you though, no matter if in a different client or through dummy. Also if you don't have a dummy connected, the Debug Prediction Margin completely freezes when you're typing in chat, no matter the action. 

This whole thing might be because Unique servers are set to the version **0.6/0,7**, but I'm not too sure.
